### PR TITLE
Fix missing `image` filter

### PR DESCRIPTION
**Pull request by jeremy**
And add specs demonstrating that all filters render their corresponding tag.

### CODE REVIEW

File: lib/jekyll/assets/liquid/filters.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ module Assets
     module Liquid
       module Filters
         ACCEPTABLE_FILTERS = [:css, :img, :asset_path, :stylesheet,
-          :javascript, :style, :img, :js]
+          :javascript, :style, :image, :js]
```
Comment (envygeeks) [inline lib/jekyll/assets/liquid/filters.rb]: I'm fine if you wish to add "image", I am not fine with the removal of "img", please keep both.  Thanks!

Comment (jeremy) [inline lib/jekyll/assets/liquid/filters.rb]: Note that `img` was not removed. It was duplicated.

Comment (envygeeks) [inline lib/jekyll/assets/liquid/filters.rb]: So it seems it was, thanks for pointing that out.

**Comment by envygeeks** [comment]
Actually, now I have the question as to why the tests needed to change?

**Comment by envygeeks** [comment]
Actually @jeremy I'm cool with that change, as long as that renderer class is moved into it's own file.  After that I'm happy to merge this changed.  Thanks!

**Comment by jeremy** [comment]
> Actually, now I have the question as to why the tests needed to change?

To introduce coverage for these filters. I may have missed existing coverage! Not sure what changed.

> as long as that renderer class is moved into it's own file.

I can make that happen. However, this is a utility class that supports this particular spec; it's not broadly useful to other specs. Scoping to the file communicates that well. Many specs define local helper methods, like '#render`, `#stub_tag`, `#fragment`, etc. This helper class fills the same role as these helper methods and is scoped accordingly.

**Comment by envygeeks** [comment]
Hey, it seems this pull request got forgotten but I would like to merge it if it still stands to fix a bug.  Can you please rebase so that I can merge this pull request after another review?

**Comment by jeremy** [comment]
Rebased 👍

**Comment by jeremy** [comment]
@envygeeks Rebased 👍

**Comment by envygeeks** [comment]
I'll make sure to re-review in the morning, thanks!

**Comment by envygeeks** [comment]
I didn't get to this today, but I'm updating to let you know I didn't forget.

**Comment by envygeeks** [comment]
Thanks :heart:

**Pull request merged**